Tribler/Test/test_as_server.py

```diff
@@ -260,7 +260,6 @@ def setUpPreSession(self):
         self.config.set_videoplayer(False)
         self.config.set_enable_metadata(False)
         self.config.set_upgrader_enabled(False)
-        self.config.set_http_api_enabled(False)
```

Comment:
leave these lines, we really want this to stay disabled on the tests even when the REST iface gets enabled by default in the future. (in tunnel/main.py too)